app/controllers/publish/providers/study_sites_controller.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+
+module Publish
+  module Providers
+    class StudySitesController < PublishController
+      before_action :pundit
+      before_action :site, only: %i[show delete]
+      before_action :build_study_site, only: %i[new create]
+
+      def index
+        @study_sites = provider.study_sites.sort_by(&:location_name)
+      end
+
+      def show; end
+
+      def new
+        @study_site_form = ::Support::SchoolForm.new(provider, @site, params: gias_school_params)
```

Comment:
Same here, we're mixing the support and publish `SchoolForm`. Should probably use the publish one?

Reply:
No we dont validate the code on update, it is set on create only

Reply:
So do we still need to use the support one?

Reply:
am looking at it